test/dlc_tests/ec2/pytorch/inference/test_pytorch_inference.py

```diff
@@ -119,7 +119,7 @@ def test_pytorch_inference_torchdata_gpu(
     _, image_framework_version = get_framework_and_version_from_tag(pytorch_inference)
     if test_utils.is_image_incompatible_with_instance_type(pytorch_inference, ec2_instance_type):
         pytest.skip(f"Image {pytorch_inference} is incompatible with instance type {ec2_instance_type}")
-    if Version(image_framework_version) in SpecifierSet(">=1.11,<=1.13"):
+    if Version(image_framework_version) in SpecifierSet(">=1.11,<=1.14"):
```

Comment:
In my previous comment, when I said upper bound, I meant `">=1.11,<1.14"` - let's remove the less than or equal to, even though we don't expect there to be a PT 1.14

Reply:
Updating